### ISSUE TITLE
Structural Differences between tutorial series tags and folder names

### ISSUE THREAD

**Issue by lwasser** [NEON-WorkWithData/neon-data-institute-2016#30]
Hi @mjones01 I've just noticed that we haven't been consistent with our tutorial series tags compared to our directory structure in git. 

I've gone ahead and fixed pre institute wk 3 - the new tutorial series tag is:

```
pre-institute3-rmd
```

I've renamed the images folder - pre-isntitute3-rmd and also the posts dir as well. 
I'd like to do the same for pre institute wk 1 and 2. 
While you're working on wk 2 will you please implement this as well?

I propose we use
wk1: pre-institute1-intro
wk2: pre-institute2-git
wk3: pre-institute3-rmd

Let me know if you have any questions.

**Comment by mjones01** [comment]
Sounds good - I'll add to all Week 2 Git tutorials.

**Comment by mjones01** [comment]
Just to clarify, this applies to `mainTag` and `tutorialSeries`, correct.

**Comment by neondataskills** [comment]
Wonderful. Thank you!
I'll go through and update the other days as I work on them (Mon-Friday), etc.

**Comment by lwasser** [comment]
Yes, it applies to both. I pushed up the RMD edits so you can see what i did there. Ii've already updated the yaml (data and org folders) with wk 2 and 3 changes. i'll hold off on wk0 and 1 until i have more time to test it given we've already sent that out.
